### ISSUE TITLE
Symlinked directories in project are missing

### ISSUE THREAD

**Issue by lassizci** [ansible/awx#10308]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
AWX deletes symlinked directories and overall results in unclean git repo.

##### ENVIRONMENT
* AWX version: 19.1.0
* AWX install method: minikube
* Ansible version:  2.11

##### STEPS TO REPRODUCE

Have a repo structure:
```
project/
 shared_vars/
    some_dir/
      vars1.yml
      vars2.yml
  inventories/
    prod/
      vars/
        some_dir -> ../../../shared_vars/some_dir
    dev/
      vars/
        some_dir -> ../../../shared_vars/some_dir
```

Launch a job.

##### EXPECTED RESULTS
* `project/inventories/prod/vars/some_dir/vars1.yml` should exist.
* `project/inventories/dev/vars/some_dir/vars1.yml` should exist.

Generally expecting to have a project directory that's exactly like the repository I get when I clone (recursing submodules).

##### ACTUAL RESULTS
* Symlinked directories are deleted
* All the symlinks have type changes, which makes the repo look like unclean

##### ADDITIONAL INFORMATION
I'm also using git-crypt. I have inventory plugin which essentially just runs `git-crypt unlock` to make the decryption transparent. This however expects there are no local changes, so it can't really work. Even I managed to run `git checkout -f .`, my submodules would still be unclean.

**Comment by shanemcd** [comment]
This was resolved in a PR yesterday and will be out in a release early next week.

**Comment by Seb0042** [comment]
Hi,

I have the same issue with AWX 19.2.0.

**Comment by wenottingham** [comment]
This is fixed in 19.2.1 + the latest awx-ee.
